docs/developer-docs/identity/verifiable-credentials/overview.mdx

```diff
@@ -0,0 +1,51 @@
+---
+keywords: [intermediate, concept, identity, credentials, verifiable credentials]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+A credential is a document or certificate that validates an individual's identity or qualifications. Credentials are a part of daily life, with common examples including:
+
+- A driver's license that qualifies an individual to operate a motor vehicle.
+- A university degree that can be used to assert a level of education.
+- A government-issued passport enabling travel between countries.
+
+These are examples of physical credentials, as they have a physical component that must be presented for validation. Credentials can also be digital, relying on authentication using cryptographic signatures such as canister signatures on ICP.
+
+Verifiable credentials (VCs) are a type of digital credential that enables individuals, organizations, or things (such as devices) cryptographically.
+
+The technical implementation of VCs on ICP can be found in the [specification](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md). It is recommended you first read the rest of this documentation, as it includes technical details.
+
+## Benefits
+
+Verifiable credentials are designed to be tamper-evident, privacy-preserving, and interoperable across different systems and contexts. By using verifiable credentials, you will have a mechanism to express these sorts of credentials on ICP in a way that is cryptographically secure, privacy-preserving, and machine-verifiable.
+
+## Terminology
```

Comment:
Should these terms be updated just in this overview page where they are defined, or across all docs and the VC spec?

Reply:
Ideally they would be updated everywhere. But since this is quite a big step to take, let me discuss this with @lmuntaner first. I'll report back.

Reply:
@jessiemongeon1: I quickly discussed with @przydatek and @lmuntaner. The terminology varies depending on the documents / context.

I think it would be best to just expand the definition of the relying party to include:

> The relying party is must verify the received credentials and hence is also commonly referred to as "verifier".

(or some other wording that you prefer)

Reply:
Revised the relying party definition to include "Since the relying party must verify the received credentials, it is also commonly referred to as the "verifier".